### DISCUSSION TITLE
Static Virtual IP for each user

### DISCUSSION THREAD

**Discussion by adke29** [strongswan/strongswan#1994] [Q&A]
Hello there, I want to ask if is there any configuration that enables server to assign unique virtual IP for each user so that every time user reconnects they will get the same IP address

**Comment by tobiasbrunner** [comment]
Please have a look at [the different virtual IP backends](https://docs.strongswan.org/docs/5.9/features/vip.html#_responder_configuration). Some support assigning static IPs based on client identities.

**Reply by f1-outsourcing** [reply]
I don't really get from this page how to do this.

**Comment by f1-outsourcing** [comment]
From a few clients that connect I would like to assign one client a static ip address. Like mentioned in this question[1]. Should I copy connection profiles and add something connection.xxx.remote.id in one and put different pool? 

Or is there a smarter solution in the pools section uses ids?





[1]
https://serverfault.com/questions/967375/how-can-i-assign-static-ips-to-strongswan-clients-based-on-their-certificates

**Comment by f1-outsourcing** [comment]
maybe it is good to support in dhcp vendor-class-identifier so I can at least identify on the dhcp server what origin these requests have.
